### PR TITLE
Con/Decon Part 1

### PR DESCRIPTION
**Pull request by Kurfursten**
- Includes construction/deconstruction for ATMs, requests consoles, and newscasters.
- ATMs are not too fancy, just make em with metal and take em apart with a wrench
- Newscasters can be taken down with a screwdriver to get a frame you can put up anywhere. Extras found in the Engi-Vend.
- Requests Consoles need to visit the new config screen (requires engineering access) to disable the lock and take down. You can also use this screen to name the department and set department type.

### CODE REVIEW

**Comment by PJB3005** [comment]
I recommend doing that config screen with multitool menus, actually

File: code/game/machinery/requests_console.dm [hunk 1]
```diff
@@ -375,6 +406,11 @@ var/list/obj/machinery/requests_console/allConsoles = list()
 				icon_state="req_comp_rewired"
 	if (istype(O, /obj/item/weapon/screwdriver))
 		if(open)
+			if(prob(50))
```
Comment (ComicIronic) [inline code/game/machinery/requests_console.dm]: Well this is just rude.

File: code/game/machinery/requests_console.dm [hunk 2]
```diff
@@ -382,16 +418,29 @@ var/list/obj/machinery/requests_console/allConsoles = list()
...
+		qdel(src)
+		return
 	if (istype(O, /obj/item/weapon/card/id) || istype(O, /obj/item/device/pda))
+		if(screen == 5)
+			var/obj/item/weapon/card/id/ID = O.GetID()
+			if (access_engine_equip in ID.access || hackState == 1)
+				announceAuth = 1
+			else
+				announceAuth = 0
+				user << "\red You are not authorized to configure this panel."
```
Comment (ComicIronic) [inline code/game/machinery/requests_console.dm]: No color

File: code/game/objects/items/stacks/sheets/sheet_types.dm [hunk 3]
```diff
@@ -78,6 +78,8 @@ var/global/list/datum/stack_recipe/metal_recipes = list ( \
 	new/datum/stack_recipe("fire alarm frame", /obj/item/mounted/frame/firealarm, 2), \
 	new/datum/stack_recipe("lightswitch frame", /obj/item/mounted/frame/light_switch, 2), \
 	new/datum/stack_recipe("intercom frame", /obj/item/mounted/frame/intercom, 2), \
+	new/datum/stack_recipe("requests console", /obj/machinery/requests_console, 4), \
```
Comment (Kurfursten) [inline code/game/objects/items/stacks/sheets/sheet_types.dm]: Requests console is only 2

File: code/game/machinery/requests_console.dm (hunk 1 above)
Comment (d3athrow) [inline code/game/machinery/requests_console.dm]: For what purpose

**Comment by d3athrow** [comment]
Please apply the commented changes and i can pull.

File: code/game/machinery/requests_console.dm (hunk 1 above)
Comment (Kurfursten) [inline code/game/machinery/requests_console.dm]: I think the original had a shock chance, but if you don't like it I'll take it out.

Comment (ComicIronic) [inline code/game/machinery/requests_console.dm]: I don't see a shock chance at all in the edited code.

**Pull request merged**